README.md

```diff
@@ -172,11 +172,18 @@ danger is a new fingerprinting surface. To remediate this:
 - This should be available only when the document is in focus (same as `navigator.clipboard.read()`).
 - The ID returned by this should be unique to the origin calling the method and change every time the site data for it
   is deleted.
+- The returned promise should be rejected before the document first successfully calls `read()` or `readText()` within a
+  session. In particular, a browser that implements persistent clipboard permissions might just put this behind
+  `clipboard-read`.
 
 In this way, correlation of users cross-site should be impossible based on either the number itself or the exact timing
 of this number changing. Hence, this API should not provide any substantially new information to the site except a hint
 when to best call `read()` so that it's optimal and user-friendly.
 
+The last consideration will limit any attempts of using this outside of the intended use case. The driver behind this
+is - sites that do not read from the clipboard should never need to use this and doing so would probably be some attempt
```

Comment:
This could be a bit shorter:

```suggestion
The last consideration will limit any attempts of using this outside of the intended use case. 
Sites that do not read from the clipboard should never need to use this and doing so would probably be some attempt
```